src/Microsoft.DotNet.Interactive.Http/HttpKernel.cs

```diff
@@ -318,6 +321,209 @@ private List<Diagnostic> GetAllDiagnostics(HttpRequestParseResult parseResult)
             return node.CreateBindingSuccess(value);
         }
 
-        return node.CreateBindingFailure(HttpDiagnostics.CannotResolveSymbol(variableName));
+        var bindingResult = MatchExpressionValue(node, expression);
+        if ( bindingResult is not null)
+        {
+            return bindingResult;
+        } else
+        {
+            return node.CreateBindingFailure(HttpDiagnostics.CannotResolveSymbol(variableName));
```

Comment:
Since we now support more than just simple variables / symbols inside embedded expression blocks, perhaps we should rename `HttpDiagnostics.CannotResolveSymbol` to `HttpDiagnostics.UnableToEvaluateExpression`. Could you please update the function name (and update the diagnostic message inside the function to match the new name)?